data/Thomas_2017/data.csv

```diff
@@ -1,19 +1,195 @@
-Abbreviation,Yrs,Ht,Length,Width,Diam,SLA,SD,SD_MASS,N,Family,Genus,species,species_binomial,site
```

Comment:
This file has many more rows now, is that correct?

Reply:
Quite a bit of the data had been cut out during the previous processing and I added it back in.